### ISSUE TITLE
extension add command fails on a current install

### ISSUE THREAD

**Issue by kwalk123** [ServiceNow/servicenow-cli#4]
When running command "snc extension add --name ui-component" on a fresh install with current Node.js with its included npm, ServiceNow CLI gives an error: "NPM version 10.2.3 found, expected >= 6.13.4". As we can see, 10.2.3 is >= to 6.13.4, and so it should run without error. Other commands such as " snc extension list-available -o table" work. I am on Windows 11 Pro 22H2 build 22621.2428. Using Terminal within Visual Studio Code, and same error when using the basic cmd. ServiceNow CLI 1.1.0; Node.js 21.1.0; and npm 10.2.3.

**Comment by mmeyer429** [comment]
Be aware of bad urls no way get a link to the newest doc since the release is hardcoded in the url:  

https://developer.servicenow.com/dev.do#!/reference/next-experience/washingtondc/cli/getting-started

Tldr; ServiceNow CLI currently supports Node version v14 / Please note that the ServiceNow CLI supports npm version 6.13.4.

I recommend installing nvm, then install node 14 LTS version with: `nvm install lts/fermium` and use a `.nvmrc` file to be able to swap to that version with `nvm use`.

Also avoid google and "docs.servicenow" they keep pointing to outdated documentation even for the current releases.
